OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
+- brendandburns
+- deads2k
```

Comment:
Please remove deads2k and lavalamp from both lists. they've never contributed here.